worlds/stardew_valley/test/TestCrops.py

```diff
@@ -8,13 +8,12 @@ class TestCropsanityRules(SVTestBase):
     }
 
     def test_need_greenhouse_for_cactus(self):
-        harvest_cactus = self.world.logic.region.can_reach_location("Harvest Cactus Fruit")
-        self.assert_rule_false(harvest_cactus, self.multiworld.state)
+        self.assert_location_cannot_be_reached("Harvest Cactus Fruit")

```

Comment:
I think the location name should still be in a variable, even if we don't need the actual rule for it